tests/test_trigger_policy.py

```
@@ -0,0 +1,597 @@
+        result = detect_trigger(
+            "John 3:16 esv", None, TriggerMode.DIRECT_ONLY, "!bible", "@bot:x", "kjv"
+        )
+        assert result is not None
+        assert result.translation == "esv"
+
+    def test_verse_range(self):
+        result = detect_trigger(
+            "1 Cor 15:1-4", None, TriggerMode.DIRECT_ONLY, "!bible", "@bot:x", "kjv"
+        )
+        assert result is not None
+        assert "1 Corinthians 15:1-4" == result.passage
```

Ruff (0.15.10)

[warning] Yoda condition detected

Rewrite as `result.passage == "1 Corinthians 15:1-4"`

(SIM300)